app/generate/page.tsx

```
@@ -0,0 +1,118 @@
+    }
+    
+    setLoading(false);
+  };
+
+  return (
+    <div className="max-w-4xl mx-auto p-6">
+      <h1 className="text-2xl font-bold mb-6">Generate Job Description</h1>
+      
+      <form onSubmit={handleSubmit} className="space-y-6">
+        <div>
+          <label className="block font-medium mb-1">Job Title</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)